source/index.html.md

```diff
@@ -21,50 +19,20 @@ code_clipboard: true
 
 # Introduction
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+Welcome to the Float Market API! You can use our API to access Float Market API endpoints, which can get information on various listings.
```

Comment:
Should probably make this generic "CSGOFloat API" that comprises the market, inspect, etc...